Sources/MongoSwift/MongoCollection+BulkWrite.swift

```diff
@@ -0,0 +1,131 @@
+import libmongoc
+
+/// An extension of `MongoCollection` encapsulating bulk write operations.
+extension MongoCollection {
+    /**
+     * Sends a batch of writes to the server at the same time.
```

Comment:
Not sure if it's worth changing this, but "at the same time" may not be technically correct since operations will be sent in batches. For PHPLIB, we simply say "Executes multiple write operations" and the logic for splitting requests into batches is discussed elsewhere.